test/config_cat/client_test.exs

```diff
@@ -3,29 +3,28 @@ defmodule ConfigCat.ClientTest do
 
   import Mox
 
-  alias ConfigCat.{Client, Constants, MockCachePolicy}
-
-  require ConfigCat.Constants
+  alias ConfigCat.{Client, MockCachePolicy}
 
   @cache_policy_id :cache_policy_id
 
   setup :verify_on_exit!
 
   setup do
-    feature = "FEATURE"
-    value = "VALUE"
-    variation = "VARIATION"
-
-    config = %{
-      Constants.feature_flags() => %{
-        feature => %{
-          Constants.variation_id() => variation,
-          Constants.value() => value
+    config = Jason.decode!(~s(
+      {
+        "p": {"u": "https://cdn-global.configcat.com", "r": 0},
```

Comment:
I thought you wanted to use the constants here 😅

Reply:
Yeah, I did. I copied this straight from the Ruby tests and am still debating with myself about whether to convert it from the raw JSON to an Elixir map using the constants.

I prefer the explicitness of using the constants, but there's also an argument to be made that it's easier to copy the same JSON blob between all of the different language SDKs when updating something in the config format.

I'll think about it a bit more before merging this.

Reply:
To be honest, I prefer it as you did. Changes like this aren't supposed to happen without breaking the tests so I prefer to explicitly fix the mocks instead of making them "smart" and something goes unnoticed. 😅

Reply:
OK, I'll make the change and get you to re-review before I merge this PR, then.

Reply:
Per further discussion on Slack, we decided to keep the JSON here to make it easier to update tests in all of the SDKs in the future, so merging as-is.